src/javascripts/test/unit/Crud/fieldView/BooleanFieldView.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('chai').assert;
```

Comment:
Wanted to be sure of the validation.required checks in `BooleanFieldView.getWriteWidget`